config/default.schema.json

```diff
@@ -428,6 +428,11 @@
 			"default": null,
 			"description": "The url of status message provider (should end without a slash)."
 		},
+		"DISPLAY_INCIDENTS_IN_MIN": {
```

Comment:
I find the name a bit confusing....should refer to alert service
SHOW_ALERTS_IN_MIN or something like that?